docs/integrations/data-ingestion/clickpipes/mysql/datatypes.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 'ClickPipes for MySQL: Supported data types'
+slug: /integrations/clickpipes/mysql/datatypes
+description: 'Page describing MySQL ClickPipe datatype mapping from MySQL to ClickHouse'
+---
+
+Here is the supported data-type mapping for the MySQL ClickPipe:
+
+| MySQL Type                                                                 | ClickHouse type                             | Notes                                                                                  |
+| -------------------------------------------------------------------------- | ------------------------------------------ | -------------------------------------------------------------------------------------- |
+| Enum                                                                       | LowCardinality(String)                     | MySQL doesn't have user-defined types for enums; instead, columns have type `enum('a','b','c')`. |
+| Set                                                                        | String                                     | `set` is like `enum`, except from `set('a','b','c')` value can be `'a,b'`. `set` can only have 64 items, as it's internally a 64-bit bitset. |
+| Decimal                                                                    | Decimal                                   | `numeric` in MySQL is the same as `decimal`, with a maximum of 65 digits. Could use more boundary checking. |
+| TinyInt                                                                    | Int8                                      | Supports unsigned.                                                                     |
+| SmallInt                                                                   | Int16                                     | Supports unsigned.                                                                     |
+| MediumInt, Int                                                             | Int32                                     | Supports unsigned.                                                                     |
+| BigInt                                                                     | Int64                                     | Supports unsigned.                                                                     |
+| Year                                                                       | Int16                                     | This type is limited; it only supports `0000` and `1900` to `2155`.                    |
```

Comment:
```suggestion
| Year                                                                       | Int16                                     |                    |
```
note left for team's sake, customer should be aware of YEAR limitations if they're using this bad type